src/py42/services/detectionlists/high_risk_employee.py

```diff
@@ -91,7 +93,14 @@ def remove(self, user_id):
             u"userId": user_id,
         }
         uri = self._make_uri(u"/remove")
-        return self._connection.post(uri, json=data)
+        try:
+            return self._connection.post(uri, json=data)
+        except Py42NotFoundError as err:
+            message = (
+                "User {} is not currently on the high-risk-employee "
```

Comment:
Would be better if it were `User with ID '{}' is not ... `

That way, it matches the other exceptions and reads more truthfully

Reply:
Changed.